.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: python
 
 python:
-    - "2.7"
-    - "3.4"
```

Comment:
If 3.4 support is removed because it's being deprecated it should be documented in the changelog and likewise the 3.4 classifier removed.

Reply:
I think we should just deprecate all the versions that travis isn't testing on... realistically speaking, people only use 3.7, and rarely 3.6 or 2.7

Reply:
but can you please handle this part, it's best that you choose what's still supported